packages/contracts/contracts/DependencyRegistryV0.sol

```diff
@@ -727,10 +776,31 @@ contract DependencyRegistryV0 is
             dependencyDetails[_dependencyType].scriptBytecodeAddresses[_index];
     }
 
+    /**
+     * @notice Returns the storage library version for
+     *         dependency `_dependencyTypes` at script index `_index`.
+     */
+    function getDependencyScriptBytecodeStorageVersionAtIndex(
+        bytes32 _dependencyType,
+        uint256 _index
```

Comment:
nit: some places we call this `scriptId`, other places we call it `index`. probably should choose one or the other (I don't have strong opinion either way!)

Reply:
Good call out – going to go with `index` throughout (updated in recent commit), as this is a simpler language to use consistently across a) dependencies, b) repository URLs, and c) CDN URLs, rather than using varied nomenclature throughout. 

This departs a bit from how we refer to the script indexes on CoreV3 (where we use `scriptId`), but I think this is the preferable trade-off. 

That seem right to you @ryley-o ?

Note: I've done a broader pass over the entirety of our implementation here around this to try to improve consistency and readability, so an additional overall review pass would be much appreciated.

Reply:
sounds good to me! will cruise through another review 👍